user-documentation/moderne-cli/how-to-guides/clone-and-sync.md

```diff
@@ -0,0 +1,109 @@
+This is the equivalent of what `git clone` would do for each of the repositories. 
+Followed by checking out the commit (and branch) of the latest LST present in your Moderne instance.
+
+{% hint style="info" %}
+The **mod git clone moderne** command does not load LSTs and a subsequent **mod build** command is required to be able to run recipes.
+{% endhint %}
+
+### Options
+You can pass the following clone options into the clone command
+
+- `--filter` - This is the equivalent to the `git clone --filter` option. [Read more](https://git-scm.com/docs/git-clone/en#Documentation/git-clone.txt-code--filtercodeemltfilter-specgtem)
+- `--depth` - This is the equivalent to the `git clone --depth` option. [Read more](https://git-scm.com/docs/git-clone/en#Documentation/git-clone.txt-code--depthcodeemltdepthgtem)
+- `--single-branch` - Only clone the branch Moderne knows for this repository
+- `--limit` - Only clone the first n repositories. Usage is not recommended outside of debugging purposes.
+- `--metadata` or `--metadata-only` - Make a [metadata only clone](#metadata-only-clone)
+
+### Directory structure
+
+Repositories are cloned into directories by their path. What the path looks like depends on the SCM the code is stored in. For instance in GitHub the code path looks like `organization/repository` or `openrewrite/rewrite-spring` while in GitLab it can have more path segments `parent-group/child-group/repository`.
+
+As an example, the structure for the `Default` organization on our app-tenant looks like this:
+```
+.
+└── clone_into
+    ├── Netflix
+    │   ├── ndbench
+    │   ├── photon
+    │   └── ribbon
+    ├── apache
+    │   └── maven-doxia
+    ├── aws
+    │   └── amazon-documentdb-jdbc-driver
+    ├── awslabs
+    │   └── aws-saas-boost
+    ├── finos
+    │   ├── messageml-utils
+    │   ├── spring-bot
+    │   ├── symphony-bdk-java
+    │   └── symphony-wdk
+    ├── openrewrite
+    │   ├── rewrite-recipe-bom
+    │   └── rewrite-sql
+    └── spring-projects
+        ├── spring-data-commons
+        └── spring-petclinic
+```
+
+### Metadata only clone
+If you are only interested in running recipes on LSTs you can use a metadata only clone. This will only download limited metadata for each repository to be used by the `mod build` command to download the correct LST.
+
+This type of clone is much faster than a full clone, but has the downside that it does not allow applying recipe results. 
+If you find yourself in a situation that you changed your mind and you do want to apply results you need to [convert a metadata clone into a full clone](#convert-a-metadata-only-clone-into-a-full-clone)
+
+## Organization context
+Aside from repositories, `mod git clone moderne` also stores the organization context to file. The `clone_into/.moderne/organization.yml` file contains information about the cloned organization that can be used by subsequent commands. 
```

Comment:
I had a lot of trouble parsing this sentence. I think the following may be better (if I understood what you were going for):

```suggestion
`mod git clone moderne` creates directories containing metadata for each repository, and also stores metadata for the organization itself. The file `clone_into/.moderne/organization.yml` contains information about the cloned organization that can be used by subsequent commands. 
```